package/yast2-network.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Jan 25 13:52:26 UTC 2023 - Knut Alejandro Anderssen González <kanderssen@suse.com>
+
+- During installation, do not configure DHCP if there is some
+  active interface configured by firmware (jsc#PED-3138).
```

Comment:
Shouldn't we use the number of the Epic (PED-967)?

Reply:
I was not sure, just checked linuxrc and wicked PRs... so modifying it ..